articles/virtual-machines/virtual-machines-instancemetadataservice-overview.md

```diff
@@ -17,93 +17,94 @@ ms.date: 03/27/2017
+
+Available Preview Regions |
 ------------------------------------------------------------|
 West Central US |
 
+This table will be updated when the service preview becomes available in other regions.
 
+For trying out Instance Metadata Service, create a VM from [Azure Resource Manager](https://docs.microsoft.com/rest/api/resources/) or [Azure portal](http://portal.azure.com) in the above regions and follow the examples below.
 
-## Retrieving Instance metadata 
-
-Instance metadata is available for running VMs created/managed using [Azure Resource Manager](https://docs.microsoft.com/rest/api/resources/?redirectedfrom=MSDN). 
-Access all data categories for a virtual machine instance use the following URI
-
-```
- curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-03-01"
-```
+## Usage
 
-The default output for all Instance metadata is in json format(content type Application/json)
+### Versioning
+The Instance Metadata Service is versioned. Versions are mandatory and the current version is `2017-03-01`.
 
-## Usage Examples
-Following are set of examples and usage semantics for Instance metadata service
+> [!NOTE] 
+> Previous preview releases of scheduled events supported {latest} as the api-version. This format is no longer supported and will be deprecated in the future.
 
-### Versioning 
-Instance metadata service is versioned. Versions are mandatory and the current preview version is 2017-03-01.
+As we add newer versions, older versions can still be accessed for compatibility if your scripts have dependencies on specific data formats. However, please note that the current preview version may not be available once the service is generally available.
 
+### Using Headers
+When you query the Instance Metadata Service, you must provide the header `Metadata: true` to ensure the request was not unintentionally redirected.
 
-```
-curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-03-01"
-```
+### Retrieving metadata
 
-As we add newer versions, older versions can still be accessed for compatibility if your scripts have dependencies on specific data formats. Note, current preview version may not be available once the service is generally available.
+Instance metadata is available for running VMs created/managed using [Azure Resource Manager](https://docs.microsoft.com/rest/api/resources/). 
+Access all data categories for a virtual machine instance using the following request:
 
+    curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-03-01"
 
 ### Data output
-By default Instance metadata returns data in json (content type=application/json).Different node elements can return data in different default format as applicable, following table is a quick reference for data formats 
+By default, the Instance Metadata Service returns data in JSON format (`Content-Type: application/json`). However, different APIs can return data in different formats if requested.
```

Comment:
different APIs can be confusing , should we say different top level namespaces ?